src/components/ButtonPanel.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Button from './Button';
+import '../styles/ButtonPanel.css';
+
+const ButtonPanel = () => {
+  const white = 'rgb(224, 224, 224)';
+  const orange = 'rgb(189, 120, 8)';
+  return (
+    <div className="btn-panel">
+      <div className="btn-group">
+        <Button buttonName="AC" color={white} />
+        <Button buttonName="+/-" color={white} />
+        <Button buttonName="%" color={white} />
+        <Button buttonName="÷" color={orange} />
+      </div>
+      <div className="btn-group">
+        <Button buttonName="7" color={white} />
+        <Button buttonName="8" color={white} />
+        <Button buttonName="9" color={white} />
+        <Button buttonName="×" color={orange} />
+      </div>
+      <div className="btn-group">
+        <Button buttonName="4" color={white} />
+        <Button buttonName="5" color={white} />
+        <Button buttonName="6" color={white} />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.